src/Umbraco.Web/PublishedCache/NuCache/DataSource/CultureVariation.cs

```diff
@@ -1,35 +1,45 @@
 ﻿using System;
+using System.Runtime.Serialization;
 using Newtonsoft.Json;
 
 namespace Umbraco.Web.PublishedCache.NuCache.DataSource
 {
     /// <summary>
     /// Represents the culture variation information on a content item
     /// </summary>
+    [DataContract] // NOTE: Use DataContract annotations here to control how MessagePack serializes/deserializes the data to use INT keys
     public class CultureVariation
     {
+        [DataMember(Order = 0)]
         [JsonProperty("nm")]
         public string Name { get; set; }
 
+        [DataMember(Order = 1)]
         [JsonProperty("us")]
         public string UrlSegment { get; set; }
 
+        [DataMember(Order = 2)]
         [JsonProperty("dt")]
         public DateTime Date { get; set; }
 
+        [DataMember(Order = 3)]
         [JsonProperty("isd")]
         public bool IsDraft { get; set; }
 
         //Legacy properties used to deserialize existing nucache db entries
+        [DataMember(Order = 4)]
```

Comment:
I don't think those legacy properties should be serialized for message pack [IgnoreMember]

Reply:
yep i can remove that but they don't actually serialize when they aren't there but happy to add the attribute